server/test/services/export/AbstractExporterTest.java

```diff
@@ -703,5 +703,21 @@ public Application submit() {
 
       return application;
     }
+
+    public Application obsolete() {
```

Comment:
nit (not-blocking): Can we name this with a verb or preposition to match the other methods in this "builder"

Reply:
Also, might be helpful to have setting it obsolete be distinct from submitting it, if possible? I'm not sure how the lifecycle stage currently gets updated in prod code.

Reply:
This was my intention, invoking the arguably cringy [obsolete-as-a-verb](https://www.merriam-webster.com/dictionary/obsolete) usage.